prov/efa/configure.m4

```diff
@@ -113,6 +115,44 @@ AC_DEFUN([FI_EFA_CONFIGURE],[
 	AC_SUBST(efa_CPPFLAGS)
 	AC_SUBST(efa_LDFLAGS)
 	AC_SUBST(efa_LIBS)
+
+	AC_ARG_WITH([cmocka],
+		[AS_HELP_STRING([--with-cmocka=DIR],
+				[Provide path to where the Cmocka runtime libraries
+				 are installed.])],
+		[efa_unit_test=yes], [efa_unit_test=no])
+
+	if [test x"$efa_unit_test" = x"yes"] && [test x"$with_cmocka" != x"yes"]; then
+		FI_CHECK_PACKAGE(cmocka,
+						 [cmocka.h],
+						 [cmocka],
+						 [_expect_any],
+						 [],
+						 [$with_cmocka],
+						 [$with_cmocka/lib64],
+						 [
+							AC_DEFINE([EFA_UNIT_TEST], [1], [EFA unit testing])
+							efatest_CPPFLAGS="$efatest_CPPFLAGS $efadv_CPPFLAGS -I$with_cmocka/include"
+							efa_CPPFLAGS="$efatest_CPPFLAGS $efadv_CPPFLAGS"
+							efa_LDFLAGS="-L$with_cmocka/lib64"
+							efatest_LDFLAGS="-L$with_cmocka/lib64"
```

Comment:
As mentioned before, you do not need `efatest_CPPFLAGS` and `efatest_LDFLAGS`. You do not need to update `efa_CPPFLAGS` and `efa_LDFLAGS` either.

`FI_CHECK_PACKAGE` will find cmocka and set `cmocka_CPPFLAGS` and  `cmocka_LDFLAGS` for you.

you just use them in linking and compiling of `efa_test`

Reply:
removed.